README.md

```diff
@@ -18,10 +18,13 @@ files on top of it.
     to emit the warning `manifest_sorter.go:175: info: skipping unknown hook: "crd-install"` 
```

Comment:
@PennyScissors I manually added changes to this file and moved the README.md generated from the charts-build-scripts over to docs/README.md since you requested that we do not change the main README.md on this repository. Let me know if anything needs to be fixed here!